soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -1968,6 +1970,9 @@ void RandomizerSettingsWindow::DrawElement() {
         }
         if (randomizerPresetSelected != 0) {
             applyPreset(selectedPresetDef.entries);
+            // force excluded location list and trick list update if tab is open.
+            locationsTabOpen = false;
+            tricksTabOpen = false;
```

Comment:
I think these need to run regardless if a preset is selected. Selecting default while there's locations/tricks selected will "delete" them so the list could change.

Reply:
You're right, good catch. Should be sorted now.